server/src/main/java/org/opensearch/search/pipeline/Pipeline.java

```diff
@@ -132,7 +137,12 @@ SearchRequest transformRequest(SearchRequest request) throws SearchPipelineProce
                         request = processor.processRequest(request);
                     } catch (Exception e) {
                         onRequestProcessorFailed(processor);
-                        throw e;
+                        if (processor.getIgnoreFailure()) {
+                            logger.info("failed to process request process: " + processor.getType());
```

Comment:
IMO, this `INFO`-level log doesn't help and just risks cluttering the logs. You can include the processor type in the `WARN`-level message.